examples/digital-signature/Cargo.toml

```diff
@@ -15,6 +15,8 @@ sha2 = "0.10"
 
 [dev-dependencies]
 test-log = "0.2"
+tracing = { version = "0.1", default-features = false, features = ["attributes"] }
+tracing-subscriber = { version = "0.3", features = ["env-filter"] }
```

Comment:
Is this related?

Reply:
Yes, So... oddly enough. This PR made this example fail on `cargo test` with the following error and adding this helped the test pass. I'm not sure what exactly changed but it fails to compile without these additional deps now.
```
% cargo test
   Compiling risc0-binfmt v0.16.1 (/Users/erik/risc0/risc0/binfmt)
   Compiling risc0-build v0.16.1 (/Users/erik/risc0/risc0/build)
   Compiling risc0-zkvm v0.16.1 (/Users/erik/risc0/risc0/zkvm)
   Compiling digital-signature-methods v0.1.0 (/Users/erik/risc0/examples/digital-signature/methods)
   Compiling digital-signature-core v0.1.0 (/Users/erik/risc0/examples/digital-signature/core)
sign: Starting build for riscv32im-risc0-zkvm-elf digital-signature-methods(build), risc0-zkvm                                                                           
sign:     Updating crates.io index
sign:     Finished release [optimized] target(s) in 1.37s-signature-methods(build)                                                                                       
   Compiling digital-signature v0.1.0 (/Users/erik/risc0/examples/digital-signature)
error[E0433]: failed to resolve: could not find `tracing_subscriber` in the list of imported crates
  --> digital-signature/src/lib.rs:72:5
   |
72 |     #[test]
   |     ^^^^^^^ could not find `tracing_subscriber` in the list of imported crates
   |
   = note: this error originates in the attribute macro `test` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: could not find `tracing` in the list of imported crates
  --> digital-signature/src/lib.rs:72:5
   |
72 |     #[test]
   |     ^^^^^^^ could not find `tracing` in the list of imported crates
   |
   = note: this error originates in the attribute macro `test` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0433`.
error: could not compile `digital-signature` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```